ui/client/theme.js

```diff
@@ -111,3 +114,54 @@ export default createTheme(theme, {
     }
   },
 });
+
+const dojoTheme = {
+  custom: {
+    nav: {
+      backgroundColor: '#06B8EF',
+      image: 'linear-gradient(to right, #06B8EF, #A11BDA)',
+    },
+    landing: {
+      backgroundColor: '#06B8EF',
+      color: 'white',
+    },
+    button: {
+      color: 'white',
+      backgroundColor: 'black',
+    },
+  },
+};
+
+const ifpriTheme = {
+  custom: {
+    nav: {
+      color: '#ffb300',
+      // ifpri logo green: '#62bb46'
+      // or darker: #429d46 (from ifpri library)
+    },
+    landing: {
+      backgroundColor: '#f5f4f0',
+      color: 'black',
+    },
+    button: {
+      color: 'white',
+      backgroundColor: '#429d46',
+    },
+  },
+  palette: {
+    secondary: {
+      main: '#429d46',
+      light: '#4cb451',
+      dark: '#2e6e31',
+      constrastText: '#fff',
+    }
+  }
+};
+
+function createCustomTheme() {
+  const themeVariant = BrandSwap({ dojo: dojoTheme, ifpri: ifpriTheme });
+
+  return createTheme(deepmerge(baseTheme, themeVariant));
```

Comment:
deepmerge is the MUI utility for merging themes: [see the first example here](https://mui.com/material-ui/customization/theming/#api)